utils/sensors.py

```diff
@@ -318,7 +318,7 @@ class RTC(GenericSensor):
         'Microsecond': {
             'start': 3,
             'size': 1,
-            'conversion_function': lambda x: x*1000*1000/256.,  # ms
+            'conversion_function': lambda x: 1000000-x*1000*1000/256.,  # ms
```

Comment:
Is this a fix for the graph jumping back sometimes?

Reply:
It is 😄 The microseconds in the telemetry data are somehow decreasing